spec/models/movie_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+# RSpec.describe Movie, :type => :model do
+#   pending "add some examples to (or delete) #{__FILE__}"
+
+
+describe Movie do
+  let(:movie) { 
+    Movie.new(
+      title: 'American Sniper',
+      plot: 'From director Clint Eastwood comes \'American Sniper,\'starring Bradley Cooper as Chris Kyle, the most lethal sniper in U.S. military history. But there was much more to this true American hero than his skill with a rifle. U.S. Navy SEAL sniper Chris Kyle is sent to Iraq with only one mission: to protect his brothers-in-arms. His pinpoint accuracy saves countless lives on the battlefield and, as stories of his courageous exploits spread, he earns the nickname \'Legend.\' However, his reputation is also growing behind enemy lines, putting a price on his head and making him a prime target of insurgents. Despite the danger, as well as the toll on his family at home, Chris serves through four harrowing tours of duty in Iraq, becoming emblematic of the SEAL creed to \'leave no man behind.\' But upon returning home, Chris finds that it is the war he can\'t leave behind. (C) Warner Bros',
+      actors: 'Bradley Cooper',
+      released: Date.new(2015, 1, 16),
+      genre: 'Drama',
+      trailer: 'www.youtube.com/embed/99k3u9ay1gs',
+      earnings: 64.6
+    )
+  }
```

Comment:
Shrink this down to minimum number of lines while still fitting within a Ruler of 80